src/sparsezoo/model/model.py

```diff
@@ -78,12 +79,14 @@ def __init__(self, source: str, download_path: Optional[str] = None):
 
         if self.source.startswith(ZOO_STUB_PREFIX):
             # initializing the files and params from the stub
-            files, path, url = self.initialize_model_from_stub(self.source)
+            _model_args = self.initialize_model_from_stub(stub=self.source)
```

Comment:
well, `model_args` name actually already means something in the context of sparsezoo (inputs to the `search_model` function. Let's not use it here, ambiguous.

Reply:
changed!